firecracker/microvm.py

```diff
@@ -132,6 +135,8 @@ def prepare_jailer(self):
         system(f"chown jailman:jailman {self.jailer_path}/tmp/")
         #
         system(f"mkdir -p {self.jailer_path}/opt")
+        system(f"mkdir -p {self.jailer_path}/dev/mapper")
+        system(f"mkdir -p {self.jailer_path}/dev/aleph-mapper")
```

Comment:
Why do we prepare the dev mapper in `prepare_jailer()`? Shouldn't this be its own step?